packages/@ourworldindata/grapher/src/tooltip/TooltipProps.ts

```diff
@@ -1,18 +1,65 @@
 import { ObservableMap } from "mobx"
+import { CoreColumn } from "@ourworldindata/core-table"
+import { TickFormattingOptions } from "@ourworldindata/utils"
 
 // We can't pass the property directly because we need it to be observable.
 export interface TooltipManager {
     tooltips?: ObservableMap<TooltipProps["id"], TooltipProps>
 }
 
+export type TooltipFadeMode = "delayed" | "immediate" | "none"
+
 export interface TooltipProps {
     id: number | string
     x: number
     y: number
     offsetX?: number
     offsetY?: number
+    offsetXDirection?: "left" | "right"
     offsetYDirection?: "upward" | "downward"
-    style?: React.CSSProperties
-    children?: React.ReactNode
+    title?: string | number // header text
+    subtitle?: string | number // header deck
+    subtitleFormat?: "unit" | "notice" // optional postprocessing for subtitle
+    notice?: string | number // target year displayed in footer tolerance text
```

Comment:
Since this is always embedded in the "No data available for {notice}..." text, maybe we can find a name that is more descriptive?